libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -334,6 +334,8 @@ singleton AP_Baro method get_pressure float
 singleton AP_Baro method get_temperature float
 singleton AP_Baro method get_external_temperature float
 singleton AP_Baro method get_altitude float
+singleton AP_Baro method all_healthy boolean
+singleton AP_Baro method healthy boolean uint8_t 0 BARO_MAX_INSTANCES
```

Comment:
We get primary for all the others. Its a bit odd being able to get the health per instance but not anything else and no way to tell which is primary. So there is no way to match up the heath reading to the other values.

Reply:
Sorry i should've specified the reason. We're looking into testing using LUA thus why we are wanting to check the individual health of each sensor.